client/python/datajunction/client.py

```diff
@@ -531,6 +531,23 @@ def link_dimension_to_node(
         )
         return response.json()
 
+    def unlink_dimension_to_node(
```

Comment:
I think `unlink_dimension_from_node` makes a bit more sense, no?

Also if this is a helper function should we start adding some `_` in front for good measure?

Reply:
Good point, I changed the name and prefixed it with `_`